Create mattermost-editorial-calendar.md

aedott opened the pull request:
<!-- Thank you for contributing a pull request! Here are a few tips to help you:

1. Learn how to update the handbook: https://handbook.mattermost.com/company/how-to-guides-for-staff/how-to-update-handbook
2. If this is your first contribution, please sign the Contributor License Agreement: http://www.mattermost.org/mattermost-contributor-agreement/
   - Once signed, you will be added to the Mattermost Approved Contributor List: https://docs.google.com/spreadsheets/d/1NTCeG-iL_VS9bFqtmHSfwETo5f-8MQ7oMDE5IUYJi_Y/pubhtml?gid=0&single=true
   - If you included your mailing address, you may receive a Limited Edition Mattermost Mug as a thank you gift after your pull request is merged: https://forum.mattermost.org/t/limited-edition-mattermost-mugs/143
-->

#### Summary
<!--
A description of what this pull request does.
-->Adding a section on how to use our updated editorial calendar to the Handbook!

aedott committed:
Create mattermost-editorial-calendar.md

jasonblais reviewed (dismissed):
This is excellent, thank you! Will defer editor review to Justine.

@justinegeffen can you also work with Ashley to add this into the LHS navigation?

aedott commented:
Thanks @jasonblais! @justinegeffen should I create an additional pull request to add this page to the navigation?

jasonblais commented:
@aedott At the bottom of this page, if you click on the branch you created (`editorial-calendar`), this will open GitHub in that branch.
![Uploading image.png…]()

From there, any changes you make will be directly applied to this same pull request.

Let me know if I can help clarify :)

aedott committed:
Add editorial calendar to SUMMARY

aedott commented:
Okay! Hoping I did that correctly. @justinegeffen  I'm not sure how to resolve the conflicts -- sorry for being a Github n00b :)

justinegeffen commented:
No problem at all, @aedott! :D

justinegeffen committed:
Merge branch '0.2.1' into editorial-calendar

justinegeffen commented on SUMMARY.md:
```diff
@@ -113,6 +113,7 @@
   * [Demand Generation Reporting](operations/messaging-and-math/demand-generation-reporting.md)
   * [M&M Asana Guidelines](operations/messaging-and-math/asana-guidelines.md)
   * [Content Marketing](operations/messaging-and-math/content-marketing.md)
+    * [How to use our editorial calendar](operations/messaging-and-math/content-marketing/editorial-calendar.md)
```
```suggestion
    * [How to Use the Editorial Calendar](operations/messaging-and-math/content-marketing/editorial-calendar.md)
```

justinegeffen committed:
Update SUMMARY.md

justinegeffen commented on operations/messaging-and-math/content-marketing/mattermost-editorial-calendar.md:
```diff
@@ -0,0 +1,53 @@
+Our editorial calendar can be found in [Asana](https://app.asana.com/0/1190898966199137/board), and is used to track upcoming content to be published on the Mattermost website, blog, developer blog, and third party channels. 
+
+## Templates
+The editorial calendar includes a Templates section, with workflow templates for common content types. To create a new task using these templates, right click the Template you want to use and select "Duplicate Task." The task will automatically be added to "Inbound Topics."
```
```suggestion
The editorial calendar includes a Templates section, with workflow templates for common content types. To create a new task using these templates, right click the Template you want to use and select **Duplicate Task**. The task will automatically be added to **Inbound Topics**.
```

justinegeffen commented on operations/messaging-and-math/content-marketing/mattermost-editorial-calendar.md:
```diff
@@ -0,0 +1,53 @@
+
+## Task Workflow
+
+Every piece of content moves through the same general workflow for creation, review, scheduling and distribution, which is reflected in the "Board View" of the Asana project. 
```
```suggestion
Every piece of content moves through the same general workflow for creation, review, scheduling, and distribution, which is reflected in the **Board View** of the Asana project. 
```

jasonblais reviewed (dismissed):
:+1:

justinegeffen committed:
Update mattermost-editorial-calendar.md

Formatting changes, trailing white spaces, and contractions added. Shortened some sentences for brevity, but no content changes.
